rootfs/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM gcr.io/google_containers/ubuntu-slim:0.4
 ENV TERM=xterm
 
 # disable source repos (speeds up apt-get update)
-RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
-apt-get update && \
-apt-get install -y \
-	bash \
-	curl \
-	net-tools \
-	procps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
-bash -c "mkdir -p /usr/share/man/man{1..8}"
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list \
+	&& apt-get update \
+	&& apt-get upgrade -y \
+	&& apt-get install -y \
```

Comment:
The upstream image actually adds this into the environment so this is not necessary :) https://github.com/kubernetes/contrib/blob/42e1f2adb653ac302db10f95be1e18a1ca50c71f/images/ubuntu-slim/Dockerfile.build#L3